FileManager/ViewControllers/FilesViewController/FilesViewController.swift

```diff
@@ -0,0 +1,264 @@
+//
+//  ViewController.swift
+//  FileManager
+//
+//  Created by Екатерина Неделько on 31.05.22.
+//
+
+import UIKit
+import PhotosUI
+
+final class FilesViewController: UIViewController {
+    
+    // MARK: - Public Properties
+    
+    lazy var filesTableView: UITableView = {
+        let filesTableView = UITableView()
+        
+        filesTableView.delegate = self
+        filesTableView.dataSource = self
+        
+        filesTableView.allowsMultipleSelection = true
+    
+        filesTableView.register(FilesTableViewCell.classForCoder(),
+                        forCellReuseIdentifier: FilesTableViewCell.id)
+    
+        return filesTableView
+    }()
+  
+    lazy var filesCollectionView: UICollectionView = {
+        filesCollectionView = UICollectionView(frame: view.frame, collectionViewLayout: layout)
+        
+        filesCollectionView.delegate = self
+        filesCollectionView.dataSource = self
+        
+        filesCollectionView.allowsMultipleSelection = true
+        
+        filesCollectionView.register(FilesCollectionViewCell.self,
+                                     forCellWithReuseIdentifier: FilesCollectionViewCell.id)
+        
+        return filesCollectionView
+    }()
+    
+    let manager = FilesManagerService()
+    
+    // MARK: - Private properties
+    
+    private let layout: UICollectionViewFlowLayout = {
+        let layout = UICollectionViewFlowLayout()
+        layout.sectionInset = UIEdgeInsets(top: 10, left: 10, bottom: 10, right: 10)
+        layout.itemSize = CGSize(width: 60, height: 60)
+        
+        return layout
+    }()
+    
+    private var mainMenuItems: [UIMenuElement] {
+        return [
+            UIAction(title: "Select",
+                     image: UIImage(systemName: "checkmark.circle"),
+                     handler: { (_) in
+                         self.manager.viewMode = .select
+            }),
+            UIAction(title: "Add new folder",
+                     image: UIImage(systemName: "folder.badge.plus"),
+                     handler: { _ in
+                         self.handleNewFileButtonTap()
+            }),
+            UIAction(title: "Add new photo",
+                     image: UIImage(systemName: "photo"),
+                     handler: { _ in
+                         self.handleNewFileButtonTap(type: .photo)
+            }),
+            UIAction(title: "Upload image",
+                     image: UIImage(systemName: "rectangle.stack.badge.plus"),
+                     handler: { _ in
+                         self.handleNewFileButtonTap(type: .image)
+            }),
+            UIMenu(title: "",
+                   options: .displayInline,
+                   children: switchInterfaceItems)
+        ]
+    }
+    
+    private var switchInterfaceItems: [UIAction] {
+        return [
+            UIAction(title: ViewType.list.rawValue,
```

Comment:
It's better to create a property named `title` for `ViewType` then using `rawValue` straight away